docs/api.md

```
@@ -0,0 +1,1291 @@
+# REST API
+
+The REST APIs are undergoing active development, so some of the below
+APIs may change in time. APIs marked as (upcoming) are yet to be added.
+
+## /api/info
+
+Information about LedFx
+
+**GET**
```

Markdownlint (0.35.0)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)